src/lib/sections/Navigation/Navigation.stories.tsx

```diff
@@ -178,6 +184,24 @@ const meta: Meta<typeof Navigation> = {
               </Navigation.Item>
             </Navigation.List>
           </Navigation.Content>
+          <Navigation.Footer>
+            <Navigation.Content>
+              <Navigation.List>
+                <Navigation.Item>
+                  <Navigation.Link href="#">
+                    <Navigation.Icon name="information" />
+                    <Navigation.Label>Documentation</Navigation.Label>
+                  </Navigation.Link>
+                </Navigation.Item>
+                <Navigation.Item>
+                  <Navigation.Link href="#">
+                    <Navigation.Icon name="help" />
```

Comment:
(totally optional) In the SCSS file for the navigation, you could add:

`@include vf-p-icon-submit-bug;`

And then replace "help" here with "submit-bug"